plugins/analytics-module-ga/src/plugin.ts

```diff
@@ -15,6 +15,9 @@
  */
 import { createPlugin } from '@backstage/core-plugin-api';
 
+/**
+ * @public
+ */
 export const analyticsModuleGA = createPlugin({
```

Comment:
Didn't realize this was here tbh, we don't really need it right?

The module pattern isn't 100% established, but it generally doesn't export a plugin/router

Reply:
Yeah, good call.  I can at least get rid of the `@public`.  Should I go as far as to deprecate it?

Reply:
🤷‍♂️ &nbsp;May as well?  e9ca4ff